modules/installation-creating-gcp-worker.adoc

```diff
@@ -40,11 +40,7 @@ have to contact Red Hat support with your installation logs.
 
 .Prerequisites
 
-* Configure a GCP account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in GCP.
-* Create and configure networking and load balancers in GCP.
-* Create control plane and compute roles.
+* Ensure you defined the variables in the _Exporting common variables_ section.
```

Comment:
FYI some variables referenced in this section are not defined in "Exporting common variables" section. So?

For example,

`COMPUTE_SUBNET` , `ZONE_0`, and `ZONE_1` are defined in "Creating load balancers in GCP" section.
`CLUSTER_IMAGE` is defined in "Creating the bootstrap machine in GCP" section.

Reply:
@brendan-daly-red-hat It seems the section below still tells additional prerequisites, any idea? Thanks! 

[Optional: Adding the ingress DNS records](https://84718--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-user-infra#installation-gcp-user-infra-adding-ingress_installing-gcp-user-infra) [Installing a cluster on user-provisioned infrastructure in GCP by using Deployment Manager templates]

_Prerequisites_
- Configure a GCP account.
- Create and configure a VPC and associated subnets in GCP.
- Create and configure networking and load balancers in GCP.
- Create control plane and compute roles.
- Create the control plane machines.
- Create the worker machines.